custom/icds_reports/reports/take_home_ration.py

```diff
@@ -7,20 +7,20 @@
     THR_REPORT_DAY_BENEFICIARY_TYPE,
     THR_21_DAYS_THRESHOLD_DATE
 )
-
+from custom.icds_reports.utils import apply_exclude
 
 class TakeHomeRationExport(object):
     title = 'Take Home Ration'
 
-    def __init__(self, location, month, loc_level=0, beta=False, report_type='consolidated'):
+    def __init__(self, domain, location, month, loc_level=0, beta=False, report_type='consolidated'):
```

Comment:
This changes the function, but doesn't change any place it is called, which means this definitely did not work before, or won't work now

Reply:
You can ignore this, I see where it is added.

Reply:
Should I consider this PR unblocked from your end? @calellowitz

Reply:
offlined, and the answer was yes. So going to merge this.